console/console-init/ui/src/modules/device/components/DeviceListAlert/DeviceListAlert.test.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import { DeviceListAlert, IDeviceListAlertProps } from "./DeviceListAlert";
+
+describe("Alert for DeviceListPage", () => {
+  test("it renders an alert on top of DeviceListPage", () => {
+    const props: IDeviceListAlertProps = {
+      visible: true,
+      variant: "info",
+      title: "Run filter to view your devices",
+      description: "You have a total of 36,300 devices"
+    };
+    const { getByText } = render(<DeviceListAlert {...props} />);
+
+    const alertTitle = getByText("Run filter to view your devices");
+    const alertDescription = getByText("You have a total of 36,300 devices");
+
+    expect(alertTitle).toBeDefined();
```

Comment:
try to use toBeInTheDocument function instead of  toBeDefined. toBeDefined can cuase in case null.
expect(null).toBeDefined(); //it will pass but logically it should fail.
so try to use toBeInTheDocument that will more safe.
 expect(null).toBeInTheDocument(); //fail
 expect(alertTitle).toBeInTheDocument(); //pass